test/base-fork/InitAndUpgrade.t.sol

```diff
@@ -463,6 +475,7 @@ contract ForeignControllerInitAlmSystemSuccessTests is ForeignControllerInitAndU
     Init.CheckAddressParams   checkAddresses;
     Init.MintRecipient[]      mintRecipients;
     Init.LayerZeroRecipient[] layerZeroRecipients;
+    Init.MaxSlippageParams[]  maxSlippageParams;
```

Comment:
Should set maxSlippage params in setUp to have some contents so we can assert it in success test